### PR TITLE
todo initialized

### PR DESCRIPTION
**Pull request by Srish-tii**
Functionalities included:

User can:
* Add todo
* Delete todo

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import './App.css';
+import CreateArea from './CreateArea/CreateArea'
+
+function App() {
+  const [arr,setArr]=useState([]);
+
+  function saveNote(titled,noted){
+    setArr((prev)=>{
+    	return([...prev,{
+    	key:arr.length,
```
Comment (ats1999) [inline client/src/components/App.js:14]: Do we need to save`key` here?

File: client/src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }
+  function editNote(index,title,content){
+	console.log(index,title,content)
+  }
+
+  return (
+    <div>
+       <Header/>
+       <CreateArea forSubmit={saveNote}/>
+      	{arr.map(eachNote => (
```
Comment (ats1999) [inline client/src/components/App.js:35]: It would be better if we rename the variable `eachNote` to `note`. This makes more sense.

**Review by ats1999 — CHANGES_REQUESTED** [review]

File: client/src/components/App.js (hunk 1 above)
Comment (Srish-tii) [reply client/src/components/App.js:14]: Yes, this key is being used as the reference for the note which has to be deleted/edited

Comment (ats1999) [reply client/src/components/App.js:13]: ok

**Pull request closed without merging**